schema.sql

```diff
@@ -1,10 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    id INT GENERATED ALWAYS AS IDENTITY,
+    id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
     name VARCHAR(30),
     date_of_birth DATE,
     escape_attempts INT,
     neutered BOOLEAN,
-    weight DECIMAL
+    weight_kg DECIMAL,
+    species VARCHAR(50),
 );
```

Comment:
- [ ] [REQUIRED] Kindly note that the project requirement stated that you should update the TABLE Animals and not create a new table. To update a TABLE, you need to ALTER it and add the new COLUMN Species. Kindly read on ALTER for a better understanding.

Reply:
Hello sir, I made a mistake, that's why modified the table but I didn't create a new table. If you take a look at my `schema.sql` you would see I only have one table, not two.

Reply:
Alright @Strangeal . Kindly implement as I have recommended above and ask for a new review bro.
Happy coding 👍🏽

Reply:
Alright @Whoistolu, 
Thank you

Reply:
- [ ] @Strangeal as previously requested kindly use the alter keyword to modify your table, it's assumed the table is connected to a database and for you to change it you need to use an SQL command and not add the column manually as you have done.
Example of adding a column of type string `   ALTER TABLE users ADD username varchar(255);`